example/index.tsx

```diff
@@ -34,7 +34,7 @@ const AxisNowEpub = () => {
 const SWTest = () => {
   const [value, setValue] = React.useState<null | string>(null);
   React.useEffect(() => {
-    fetch('http://localhost:1234/samples/muse13454.json').then((res) => {
```

Comment:
Did you intend to edit the SWTest example for now? It's just a little playground for me to work on the SW basically so wouldn't be that useful to you right now I don't think

Reply:
I moved the file, so I just updated everything that referred to the old file.  I don't think this affects anything.

Reply:
Oh kk